sunpy/net/hek/tests/test_hek.py

```diff
@@ -197,3 +199,12 @@ def test_mixed_results_get_angstrom():
     result = client.search(attrs.Time(tstart, tend), attrs.hek.EventType(event_type))
     assert len(result) == 13
     assert result[0]["SOL_standard"] == 'SOL2014-10-24T20:53:46L247C106'
+
+
+@pytest.mark.remote_data
+def test_convert_time(hek_client_creator):
```

Comment:
Does it work with a return from fido?

Reply:
Yes, it does. Would we need a test for that as well?

Reply:
Sure.

Reply:
> Sure.

Where would this test go though? Should I import fido and write it here itself?

Reply:
Yeah.